tests/src/test/java/ca/bc/gov/open/jag/efiling/stepDefinitions/GetCourtLocationSD.java

```diff
@@ -0,0 +1,73 @@
+package ca.bc.gov.open.jag.efiling.stepDefinitions;
+
+import ca.bc.gov.open.jag.efiling.models.UserIdentity;
+import ca.bc.gov.open.jag.efiling.services.CourtService;
+import ca.bc.gov.open.jag.efiling.services.OauthService;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.Assert;
+
+public class GetCourtLocationSD {
+
+    private final OauthService oauthService;
+    private final CourtService courtService;
+    private UserIdentity actualUserIdentity;
+    private Response actualCourtLocationResponse;
+
+    public Logger logger = LogManager.getLogger(GetCourtLocationSD.class);
+
+    public GetCourtLocationSD(OauthService oauthService, CourtService courtService) {
+        this.oauthService = oauthService;
+        this.courtService = courtService;
+    }
+
+    @Given("valid user account {string}:{string} information is authenticated")
+    public void userInformationIsAuthenticated(String username, String password) {

```

Comment:
Should we move this to env variables since we are always using the same user ?

Reply:
or maybe there will be multiple...

Reply:
There will be one user. Will update to use as environment variable in the next PR if that's ok.